lensless/admm.py

```diff
@@ -191,11 +199,11 @@ def _image_update(self):
         # rk = self._convolver._pad(rk)
 
         if self.is_torch:
-            freq_space_result = self._R_divmat * torch.fft.rfft2(rk, dim=(0, 1))
-            self._image_est = torch.fft.irfft2(freq_space_result, dim=(0, 1))
+            freq_space_result = self._R_divmat * torch.fft.rfft2(rk, dim=(1, 2))
```

Comment:
sorry for one request here :see_no_evil: could you change `dim` to default to `(-3, -2)`?